packages/example/package.json

```diff
@@ -19,7 +18,6 @@
     "tsx": "^4.16.5"
   },
   "devDependencies": {
-    "@bufbuild/buf": "^1.35.1",
```

Comment:
@timostamm This was the only place we had `@bufbuild/buf` as a dep, which didn't feel right. I moved it into the top-level package.json since it seemed like a dev dependency for the whole repo really. Let me know if you'd rather only include it in the packages it's used.